librz/il/theory_bool.c

```diff
@@ -4,28 +4,28 @@
 #include <rz_il/rzil_opcodes.h>
 #include <rz_il/rzil_vm.h>
 
-void *rz_il_handler_b0(RzILVM *vm, RzILOp *op, RzILOpArgType *type) {
+void *rz_il_handler_bool_false(RzILVM *vm, RzILOp *op, RzILOpArgType *type) {
 	rz_return_val_if_fail(vm && op && type, NULL);
 
 	RzILBool *ret = rz_il_bool_new(false);
 	*type = RZIL_OP_ARG_BOOL;
 	return ret;
 }
 
-void *rz_il_handler_b1(RzILVM *vm, RzILOp *op, RzILOpArgType *type) {
+void *rz_il_handler_bool_true(RzILVM *vm, RzILOp *op, RzILOpArgType *type) {
```

Comment:
Please keep original operation names. If the name is unclear, it's better to add a Doxygen comment explaining it.
Refer to the names in https://github.com/BinaryAnalysisPlatform/bap/blob/master/lib/bap_core_theory/bap_core_theory_definition.ml#L49